docs/source/cloud-setup/policy.rst

```diff
@@ -24,90 +24,90 @@ SkyPilot has a :ref:`client-server architecture <sky-api-server>`, where a centr
 To deploy a admin policy, here are the steps:
 
 1. :ref:`Implement the policy in a Python package <implement-admin-policy>` or :ref:`host the policy as a RESTful server <host-admin-policy-as-server>`.
-2. Deploy the policy at the :ref:`server-side <server-side-admin-policy>` and/or the :ref:`client-side <client-side-admin-policy>`.
+2. Install the policy at the :ref:`server-side <server-side-admin-policy>` to enforce it for all users.
+3. Optionally, an admin policy can also be installed at the :ref:`client-side<client-side-admin-policy>` if it needs to access user's local environment.
 
-.. hint::
+The order of policy application is demonstrated below:
 
-    If admin policy is set at both client-side and server-side, the policy at client-side will be applied first, the mutated tasks and SkyPilot config will then be processed by the policy at server-side.
+.. figure:: ../images/admin-policy.png
```

Comment:
1. there are a bit too many colors. can we only have color for the client and server policies?
2. Can we make the font size a bit larger?
3. Also, CLI should be CLI/SDK.

Reply:
Sounds good!